bsp/m5stack_core/idf_component.yml

```
@@ -0,0 +1,21 @@
+version: "1.0.0"
+description: Board Support Package (BSP) for M5Stack Core
+url: https://github.com/espressif/esp-bsp/tree/master/bsp/m5stack_core
+
+targets:
+  - esp32
+
+tags:
+  - bsp
+
+dependencies:
+  idf: ">=5.0"
```

@Ocean-lhy I'm sorry, I missed this one. You are using new version of the I2C driver that was introduced in v5.2

Please also add 'm5stack_core' to this list https://github.com/espressif/esp-bsp/blob/master/.build-test-rules.yml#L6

```suggestion
  idf: ">=5.2"
```